packages/server/graphql/getRateLimiter.ts

```diff
@@ -1,11 +1,23 @@
-import {InMemoryRateLimiter} from './InMemoryRateLimiter'
+import {InMemoryRateLimiter, RateLimiter, StubRateLimiter} from '../utils/rateLimiters'
 
-let rateLimiter: InMemoryRateLimiter
+let rateLimiter: RateLimiter
 const getRateLimiter = () => {
   if (!rateLimiter) {
-    rateLimiter = new InMemoryRateLimiter()
+    rateLimiter = _getRateLimiter()
   }
   return rateLimiter
 }
 
+/**
+ * This function returns a stub rate limiter when testing, based on the CI
+ * environment variable.
+ *
+ * Exposed for testing. Use the default export outside of test.
+ *
+ * @private
+ */
+export function _getRateLimiter(): RateLimiter {
+  return process.env.CI ? new StubRateLimiter() : new InMemoryRateLimiter()
```

Comment:
I confirmed in the built files that we're referencing the environment variable, not building in the static value.

![Screen Shot 2022-04-18 at 09 33 44](https://user-images.githubusercontent.com/630449/163832526-4846d30d-51cc-461b-905a-19b51e8c5b2a.png)